src/icebreakers_api.rs

```diff
@@ -106,6 +123,34 @@ impl IcebreakersAPI {
 
             info!("Successfully registered with Icebreakers API.");
 
+            // In case we get a URI without a protocol (http: or https: or ws: or wss:):
+            let fallback_proto = if self.base_url.starts_with("https:") {
+                "wss:"
+            } else {
+                "ws:"
+            };
+
+            let success_response = SuccessResponse {
```

Comment:
Optional: It would be nice to log the load balancer information we got. It might help with troubleshooting once needde.

Reply:
Hmmm, but with the `access_token`? 👀

The URI will be logged soon after this:

https://github.com/blockfrost/blockfrost-platform/blob/2f4cab53186e39c3db6d775777d80a14658011ac/src/load_balancer.rs#L273